lib/internal/url.js

```diff
@@ -3,7 +3,8 @@
 const util = require('util');
 const {
   hexTable,
-  isHexTable
+  isHexTable,
+  encodeStr,
```

Comment:
Unexpected trailing comma. Remember to run lint before commit: `make lint`

Reply:
@starkwang I did it. And everything was ok:

![2018-03-02-1112-dubakqz7sa](https://user-images.githubusercontent.com/825219/36889374-ba4b26b2-1e0a-11e8-9a93-fe0161b06ae5.png)

Reply:
@starkwang we had this rule for trailing comma

```
'comma-dangle': ['error', 'only-multiline']
```

It allows using trailing comma. But maybe we should fix this rule. What do you think?

Reply:
Oh sorry, I misunderstood the meaning of `'only-multiline'`. Just ignore it.